Start of new architecture section

mtrimpe opened the pull request:
Firs start of new architecture section.

Commit:
Functionele plaat overgezet naar SVG

mtrimpe committed:
First image of new architecture

mtrimpe committed:
First image of new architecture

mtrimpe commented:
@marcdeboer Hier alvast een eerste plaatje voor de nieuwe architectuur. Nog verre van compleet.

marcdeboer commented on static/images/1.3.huidige_sitiuatie.png:
Hier moeten we nog naar kijken. In de nieuwe plaat is links en rechts niet zo makkelijk te onderscheiden, de applicatie links is weg (toch?) en de gebruiker lijkt rechtstreeks op de gateway te werken. De tokens en dynamic roles komen wat uit de lucht vallen, maar goed, er is nog geen tekst dus dat is wat kort door de bocht.

mtrimpe committed:
Improved drawings

mtrimpe committed:
Link naar standaard toegevoegd

mtrimpe committed:
First draft of new architecture section

mtrimpe committed:
First draft of new architecture section

mtrimpe committed:
Improved wording a littlebit.

mtrimpe committed:
Improved wording a littlebit.

Commit:
Toepassen sectie aangevuld

Commit:
Folders van leveranciers toegevoegd

Commit:
link naar CSS ook onder ftv

Commit:
link naar js ook onder ftv

Commit:
eerste opzet softwarekeus

Commit:
link incl /ftv

Commit:
gebroken links

Commit:
functionele beschrijving verplaatst naar 'Toepassen' sectie

Commit:
link naar kick off op RealisatieIBDS

Commit:
kleine tekstuele aanpassing

MaikelH committed:
Notulen kick-off toegevoegd

MaikelH committed:
Gebruik Github URL

Commit:
globale variabele voor BaseDir

MaikelH committed:
Aanpassingen Karin verwerken

MaikelH committed:
Conclusie toegevoegd (Michiel)

Co-authored-by: Michiel Trimpe <michiel@trimpe.nl>

MaikelH committed:
Remove draft status

rvendrig committed:
Draw.io bestanden toegevoegd

rvendrig committed:
Added Rijkshuisstijl iconen 2000-2999 diap.xml

rvendrig committed:
Added Rijkshuisstijl iconen 2000-2999.xml

rvendrig committed:
Added Rijkshuisstijl iconen 7000-9999 diap.xml

rvendrig committed:
Added Rijkshuisstijl iconen 7000-9999.xml

rvendrig committed:
Added FDS Visualisatiecomponenten diap.drawio.xml

rvendrig committed:
Added FDS Visualisatiecomponenten.drawio.xml

rvendrig committed:
Added Rijkshuisstijl iconen 3900-3948.drawio.xml

rvendrig committed:
Added Rijkshuisstijl iconen / 3900-3948.xml

rvendrig committed:
Verplaatsen van bilbliotheken naar aparte map

rvendrig committed:
Aanpassing dir structuur

mtrimpe committed:
Werkgroep pagina bijgewerkt

rvendrig committed:
svg exports van diagrams van Gerard Architecture_2025-03-26_07-58-36

Zijn per stuk zonder titel geexporteerd (lokaal in eerste instantie) naar de static/diagrams map

rvendrig committed:
Opschonen mapje met drawio-bestanden

Commit:
Waarom en voorbeelden in lijn met ontwerp

Commit:
Definities verwijderd

Commit:
Tekstuele verbeteringen

Commit:
baseDirectory voor op Gitlab

Commit:
detail

rvendrig committed:
Font en SVG archieven toegevoegd met iconen

rvendrig committed:
Aanvulling iconen

mtrimpe committed:
Bijgewerkte draw.io tekeningen

rvendrig committed:
Tabblad toegevoegd in diagrams met aanpassing

mtrimpe committed:
Federatief en logisch naar technisch beleid toegevoegd

mtrimpe committed:
Extra tekeningen toegevoegd

rvendrig committed:
update svg voor presentatiescherm

Commit:
Voorbeelden bijgewerkt

Commit:
Placeholder nieuws en activiteiten

Commit:
Nieuws en activiteiten

mtrimpe committed:
Notulen werkgroep

Commit:
afbeeldingen op home page

Commit:
Organisaties bij deelnemers

Commit:
'Waarom' review door Karin

mtrimpe committed:
Verbeteringen aan notulen werkgroep

mtrimpe committed:
Kleine tweaks en notulen aan index toegevoegd

Commit:
Review door Karin

MaikelH committed:
Setting up new theme for FTV

MaikelH committed:
Add RHC css to theme

MaikelH committed:
Revert "Setting up new theme for FTV"

This reverts commit ac230419af8d410928d0654e74ed7955ca327b71.

MaikelH committed:
Revert "Add RHC css to theme"

This reverts commit 620a1f675af0384e2654db1d53333d6f402b7a18.

mtrimpe committed:
Vragen aan selectie pagina toegevoegd
